esp/bindings/http/platform/httptransport.cpp

```diff
@@ -941,14 +941,14 @@ int CHttpMessage::startSend()
     return 0;
 }
 
-int CHttpMessage::sendChunk(const char *chunk)
```

Comment:
These changes do not seem to be used, and I doubt that including null characters in http responses is a good idea.